_includes/sidebar.html

```diff
@@ -6,7 +6,7 @@ <h3 id="suggest-this-license">Suggest this license</h3>
   <div class="repository-suggestion">
     <p>Make a pull request to suggest this license for a project that is <a href="/no-permission/">not licensed</a>. Please be polite: see if a license has already been suggested, try to suggest a license fitting for the project's <a href="/community/">community</a>, and keep your communication with project maintainers friendly.</p>
     <div class="input-wrapper">
-      <input type="text" data-license-id="{{ page.spdx-id }}" placeholder="Enter GitHub repository URL" id="repository-url" title="status" />
+      <input type="text" data-license-id="{{ page.spdx-id }}-only" placeholder="Enter GitHub repository URL" id="repository-url" title="status" />
```

Comment:
This is not used to link to SPDX, and isn't specific to _GPL. It should be reverted.